app/helpers/halala.rb

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+# frozen_string_literal: true
+class Halala
+  CURRENCY= ' SAR'
+  attr_reader :riyal
```

Comment:
This is subtly different from your previous behaviour. If you are happy for riyal to be accessible outside the class then this is fine. If you would rather it was kept confined to the class, as before, then it needs to be private.